llarp/dns/unbound_resolver.cpp

```diff
@@ -24,9 +25,18 @@ namespace llarp::dns
   void
   UnboundResolver::Reset()
   {
-    started = false;
     if (runner)
     {
+      // cancel all pending lookups
+      for (auto id : pending_resolve_jobs)
+      {
+        if (unboundContext)
+          ub_cancel(unboundContext, id);
+      }
```

Comment:
This shouldn't be necessary: `ub_ctx_delete` already cancels all in-progress async queries.  (We should still clear the pending_resolve_jobs, though).